### ISSUE TITLE
Translations from kimzeuner

### ISSUE THREAD

**Issue by FlensGo** [sander1988/Indego#243]
First of all, thank you for providing this integration.
How or where do I insert the new German translation? When I insert the new lines in the de.json the entities are no longer available. Is it better to wait for the next release?
By the way, invitations to Discord only come out sporadically and when they do, numerous security questions have to be answered each time.

**Comment by kimzeuner** [comment]
i think we are waiting for the other translations to complete before adding it to a new release.
except the new lines in de.json file you also have to add two lines to the init.py file.
Go to the "ENTITY_MOWER_STATE" and "ENTITY_MOWER_STATE_DETAIL" (should be around line 100). Here you have to add 
at ENTITY_MOWER_STATE the following line after "CONF_ATTR[],"
`CONF_TRANSLATION_KEY: "mower_state",`
and at ENTITY_MOWER_STATE_DETAIL add
`CONF_TRANSLATION_KEY: "mower_state_detail",`
so it should look like this afterwards:

```    
    ENTITY_MOWER_STATE: {
        CONF_TYPE: SENSOR_TYPE,
        CONF_NAME: "mower state",
        CONF_ICON: "mdi:robot-mower-outline",
        CONF_DEVICE_CLASS: None,
        CONF_UNIT_OF_MEASUREMENT: None,
        CONF_ATTR: ["last_updated"],
        CONF_TRANSLATION_KEY: "mower_state",
    },
    ENTITY_MOWER_STATE_DETAIL: {
        CONF_TYPE: SENSOR_TYPE,
        CONF_NAME: "mower state detail",
        CONF_ICON: "mdi:robot-mower-outline",
        CONF_DEVICE_CLASS: None,
        CONF_UNIT_OF_MEASUREMENT: None,
        CONF_ATTR: [
            "last_updated",
            "state_number",
            "state_description",
        ],
        CONF_TRANSLATION_KEY: "mower_state_detail",
    }, 
```

**Comment by FlensGo** [comment]
Thanks for the quick reply. It works now.
Still have "Sleeping": "Schläft", moved from mower_state to mower_state_detail.

**Comment by sander1988** [comment]
Thank you for your contribution. The updated translations have been merged in #241 and are on the develop branch now.

**Comment by WhyLev** [comment]
🤖 **AI Summary of the solution:**

The issue regarding the insertion of new German translations was resolved by adding specific lines to the init.py file and updating the de.json file, leading to successful integration.
